src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  switch (true) {
+    case days < 3:
+      return days * 40;
+    case days >= 3 && days < 7:
+      return days * 40 - 20;
+    case days >= 7:
+      return days * 40 - 50;
```

Comment:
Replace literals '40' and '50' with constants to give them meaningful names, like 'DAILY_RATE' and 'LONG_TERM_DISCOUNT'.